api/helpers/gamestate/check-game-state-for-win.js

```diff
@@ -16,9 +16,14 @@ module.exports = {
       description: 'GameState object',
       required: true,
     },
+    countPasses: {
```

Comment:
Let's rename it to numPasses, since that's closer to some of our other counting variable names